deploy.py

```diff
@@ -284,84 +513,219 @@ def verify_flashed_app(self, expected_app):
       app_found = expected_app in apps
     return app_found
 
+  def create_hex_file(self, dest_file):
+    # We produce an intelhex file with everything in it
```

Comment:
Could you link to the intelhex documentation?

Reply:
Done